### PR TITLE
chore: header cleanup

### PR DESCRIPTION
**Pull request by rebelchris**
## Changes

We actually removed the greeting a while ago, which makes it so that the animation never runs anyway.
By cleaning this up we actually are able to remove another dependency.

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://chore-header-cleanup.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/layout/HeaderLogo.tsx [hunk 1]
```diff
@@ -1,44 +1,22 @@
...
 }
 
-function HeaderLogo({
-  user,
-  greeting,
-  onLogoClick,
-  position,
-}: HeaderLogoProps): ReactElement {
-  const [showGreeting, setShowGreeting] = useState(false);
+function HeaderLogo({ onLogoClick, position }: HeaderLogoProps): ReactElement {
```
Comment (omBratteng) [inline packages/shared/src/components/layout/HeaderLogo.tsx:10]: Maybe we can remove this, by moving `const featureTheme = useFeatureTheme();` into Logo?
And then just use Logo instead?

Comment (omBratteng) [reply packages/shared/src/components/layout/HeaderLogo.tsx:10]: Or maybe not, it might change the colour other places.

Comment (rebelchris) [reply packages/shared/src/components/layout/HeaderLogo.tsx:10]: Yeah it's used quite widely actually (Logo) that is.
Would prefer for now to keep it  maybe?

Comment (omBratteng) [reply packages/shared/src/components/layout/HeaderLogo.tsx:10]: Yeah, let's just keep it.

File: packages/shared/src/components/layout/HeaderLogo.tsx [hunk 2]
```diff
@@ -1,44 +1,22 @@
...
   const featureTheme = useFeatureTheme();
 
   return (
     <>
```
Comment (sshanzel) [inline packages/shared/src/components/layout/HeaderLogo.tsx:14]: We should be able to remove this Fragment now 🥳

File: packages/shared/src/components/layout/MainLayoutHeader.tsx [hunk 3]
```diff
@@ -1,7 +1,6 @@
 import classNames from 'classnames';
 import React, { ReactElement, ReactNode, useCallback } from 'react';
 import dynamic from 'next/dynamic';
-import { useAuthContext } from '../../contexts/AuthContext';
```
Comment (sshanzel) [inline packages/shared/src/components/layout/MainLayoutHeader.tsx:4]: Feels like we can break this component to multiple smaller ones, though I think that is for another PR.

**Review by sshanzel — APPROVED** [review]
Removed packages are always the best to review 🚀

**Review by omBratteng — APPROVED** [review]

**Pull request merged**